RoT/Protocol/Challenge_Protocol.md

```diff
@@ -1,5688 +1,2854 @@
 
-<p id="gdcalert5" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image5.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert6">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+*   Endpoints should support EID assignment using the MCTP Set Endpoint ID
+    control message.
 
+*   Endpoints should support the MCTP Get Vendor Defined Message Support control
+    message to indicate support level for the Cerberus protocol.
 
-![alt_text](images/image5.png "image_tooltip")
+The Platform Cerberus Active RoT is always the MCTP master.  Active Component
+RoT’s can be configured as Endpoint, or Endpoint and Master.  An Active
+Component RoT Endpoint and Master should interface to separate physical busses.
+There is no requirement for master arbitration as master and slave definitions
+are hierarchically established.  The only hierarchy whereby the Active Component
+RoT becomes both Endpoint and Master is when there is a downstream sub-device,
+such as the Host Bus Adapter (HBA) depicted in the following block diagram:
 
+> TODO: Figure 6
 
-In this diagram, the HBA RoT is an Endpoint to the Platform Active RoT and Master to the downstream HBA Expanders.  To the Platform’s Active RoT, the HBA is an Endpoint RoT.   To the HBA Expanders, the HBA Controller is a Master RoT.
+In this diagram, the HBA RoT is an Endpoint to the Platform Active RoT and
+Master to the downstream HBA Expanders.  To the Platform’s Active RoT, the HBA
+is an Endpoint RoT.  To the HBA Expanders, the HBA Controller is a Master RoT.
 
-The messaging protocol encompasses Management Component Transport Protocol (MCPT) Base Specification, in relation to the MCTP SMBus/I2C Transport Binding Specification, whereby the Active Component RoT is Endpoint and the Platform’s Active RoT as Master.   
+The messaging protocol encompasses Management Component Transport Protocol
+(MCPT) Base Specification, in relation to the MCTP SMBus/I2C Transport Binding
+Specification, whereby the Active Component RoT is Endpoint and the Platform’s
+Active RoT as Master.
 
 ## Protocol Format
 
-All MCTP transactions are based on the SMBus Block Write bus protocol.  The following diagram shows MCTP encapsulated message.
-
-
-    Figure 7 MCTP Encapsulated Message
-
-
-    
-
-<p id="gdcalert6" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image6.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert7">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+All MCTP transactions are based on the SMBus Block Write bus protocol.  The
+following C-style struct shows an MCTP encapsulated message.
+
+```c
+struct MctpFrame {
```

Comment:
We should move this to a table.  It doesn't have to look like the table in the original spec, but I think some kind of table representation would be more generally consumable.